### ISSUE TITLE
No source information for T1 text symbols

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#581]
Compositor cannot determine source information for some characters in the [T1 / Cork character set](https://en.wikipedia.org/wiki/Cork_encoding), e.g. Ø (Oslash) `{\O}`.

Add support for all text symbols declared in t1enc.def:

```
\DeclareTextSymbol{\AE}{T1}{198}
\DeclareTextSymbol{\DH}{T1}{208}
\DeclareTextSymbol{\DJ}{T1}{208}
\DeclareTextSymbol{\L}{T1}{138}
\DeclareTextSymbol{\NG}{T1}{141}
\DeclareTextSymbol{\OE}{T1}{215}
\DeclareTextSymbol{\O}{T1}{216}
\DeclareTextSymbol{\SS}{T1}{223}
\DeclareTextSymbol{\TH}{T1}{222}
\DeclareTextSymbol{\ae}{T1}{230}
\DeclareTextSymbol{\dh}{T1}{240}
\DeclareTextSymbol{\dj}{T1}{158}
\DeclareTextSymbol{\guillemotleft}{T1}{19}
\DeclareTextSymbol{\guillemotright}{T1}{20}
\DeclareTextSymbol{\guilsinglleft}{T1}{14}
\DeclareTextSymbol{\guilsinglright}{T1}{15}
\DeclareTextSymbol{\i}{T1}{25}
\DeclareTextSymbol{\j}{T1}{26}
\DeclareTextSymbol{\ij}{T1}{188}
\DeclareTextSymbol{\IJ}{T1}{156}
\DeclareTextSymbol{\l}{T1}{170}
\DeclareTextSymbol{\ng}{T1}{173}
\DeclareTextSymbol{\oe}{T1}{247}
\DeclareTextSymbol{\o}{T1}{248}
\DeclareTextSymbol{\quotedblbase}{T1}{18}
\DeclareTextSymbol{\quotesinglbase}{T1}{13}
\DeclareTextSymbol{\ss}{T1}{255}
\DeclareTextSymbol{\textasciicircum}{T1}{`\^}
\DeclareTextSymbol{\textasciitilde}{T1}{`\~}
\DeclareTextSymbol{\textbackslash}{T1}{`\\}
\DeclareTextSymbol{\textbar}{T1}{`\|}
\DeclareTextSymbol{\textbraceleft}{T1}{`\{}
\DeclareTextSymbol{\textbraceright}{T1}{`\}}
\DeclareTextSymbol{\textcompwordmark}{T1}{23}
\DeclareTextSymbol{\textdollar}{T1}{`\$}
\DeclareTextSymbol{\textemdash}{T1}{22}
\DeclareTextSymbol{\textendash}{T1}{21}
\DeclareTextSymbol{\textexclamdown}{T1}{189}
\DeclareTextSymbol{\textgreater}{T1}{`\>}
\DeclareTextSymbol{\textless}{T1}{`\<}
\DeclareTextSymbol{\textquestiondown}{T1}{190}
\DeclareTextSymbol{\textquotedblleft}{T1}{16}
\DeclareTextSymbol{\textquotedblright}{T1}{17}
\DeclareTextSymbol{\textquotedbl}{T1}{`\"}
\DeclareTextSymbol{\textquoteleft}{T1}{`\`}
\DeclareTextSymbol{\textquoteright}{T1}{`\'}
\DeclareTextSymbol{\textsection}{T1}{159}
\DeclareTextSymbol{\textsterling}{T1}{191}
\DeclareTextSymbol{\textunderscore}{T1}{95}
\DeclareTextSymbol{\textvisiblespace}{T1}{32}
\DeclareTextSymbol{\th}{T1}{254}
```

**Comment by ktraunmueller** [comment]
Here's a test document:

```
\documentclass[12pt,a4paper]{article}

%%! \usepackage[utf8]{inputenc}
\usepackage{geometry}
\usepackage{lmodern}
\usepackage[T1]{fontenc}
\usepackage{textcomp}
%\usepackage{amsmath}
%\usepackage{amssymb}
%\usepackage{mathrsfs}
\usepackage{longtable}
\usepackage{graphicx}
\graphicspath{{./}}
\usepackage{csquotes}

\begin{document}

\title{T1 Encoding}

\maketitle

\section{Text Symbols}

\begin{longtable}{cl}
    \verb|\AE| & \AE \\
    \verb|\DH| & \DH \\
    \verb|\DJ| & \DJ \\
    \verb|\L| & \L \\
    \verb|\NG| & \NG \\
    \verb|\OE| & \OE \\
    \verb|\O| & \O \\
    \verb|\SS| & \SS \\
    \verb|\TH| & \TH \\
    \verb|\ae| & \ae \\
    \verb|\dh| & \dh \\
    \verb|\dj| & \dj \\
    \verb|\guillemotleft| & \guillemotleft \\
    \verb|\guillemotright| & \guillemotright \\
    \verb|\guilsinglleft| & \guilsinglleft \\
    \verb|\guilsinglright| & \guilsinglright \\
    \verb|\i| & \i \\
    \verb|\j| & \j \\
    \verb|\ij| & \ij \\
    \verb|\IJ| & \IJ \\
    \verb|\l| & \l \\
    \verb|\ng| & \ng \\
    \verb|\oe| & \oe \\
    \verb|\o| & \o \\
    \verb|\quotedblbase| & \quotedblbase \\
    \verb|\quotesinglbase| & \quotesinglbase \\
    \verb|\ss| & \ss \\
    \verb|\textasciicircum| & \textasciicircum \\
    \verb|\textasciitilde| & \textasciitilde \\
    \verb|\textbackslash| & \textbackslash \\
    \verb|\textbar| & \textbar \\
    \verb|\textbraceleft| & \textbraceleft \\
    \verb|\textbraceright| & \textbraceright \\
    \verb|\textcompwordmark| & \textcompwordmark \\
    \verb|\textdollar| & \textdollar \\
    \verb|\textemdash| & \textemdash \\
    \verb|\textendash| & \textendash \\
    \verb|\textexclamdown| & \textexclamdown \\
    \verb|\textgreater| & \textgreater \\
    \verb|\textless| & \textless \\
    \verb|\textquestiondown| & \textquestiondown \\
    \verb|\textquotedblleft| & \textquotedblleft \\
    \verb|\textquotedblright| & \textquotedblright \\
    \verb|\textquotedbl| & \textquotedbl \\
    \verb|\textquoteleft| & \textquoteleft \\
    \verb|\textquoteright| & \textquoteright \\
    \verb|\textsection| & \textsection \\
    \verb|\textsterling| & \textsterling \\
    \verb|\textunderscore| & \textunderscore \\
    \verb|\textvisiblespace| & \textvisiblespace \\
    \verb|\th| & \th \\
\end{longtable}

\end{document}
```

**Comment by ktraunmueller** [comment]
Fixed in 1.23

**Comment by ktraunmueller** [comment]
Some symbols, like `$` or `\guillemotleft` still aren't working, reopening.

**Comment by ktraunmueller** [comment]
Fixed in 1.24, this time for real.
